.rubocop.yml

```diff
@@ -209,6 +210,7 @@ Metrics/LineLength:
   Description: 'Limit lines to 80 characters.'
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#80-character-limits'
   Max: 80
+  Enabled: false
```

Comment:
that still resulted in many errors. I am totally on board with doing it, just felt like a lot of work for today when there were other cleanup tasks to tackle. 

I think a good strategy that ensures we actually do this eventually would be to add this back in as its own task: https://trello.com/c/8MHQSLAV